src/EntityGraphQL/Schema/BaseField.cs

```diff
@@ -64,6 +65,39 @@ protected BaseField(ISchemaProvider schema, ISchemaType fromType, string name, s
             AddValidator<DataAnnotationsValidator>();
         }
 
+        public void ApplyAttributes(IEnumerable<Attribute> attributes)
+        {
+            if (attributes.Any())
+            {
+                foreach (var attribute in attributes)
+                {
+                    if (attribute is ArgumentValidatorAttribute validator)
```

Comment:
Any reason this needs to be a special case and not just another `extension.ApplyExtension(this)`?

Reply:
isn't that fine? The directive just doesn't override the type method?